### PR TITLE
Developer

### PR DESCRIPTION
**Pull request by MusaZwane**
I have added some modification on the compose file( volumes and env ) . New needed directories.
Added a certs directory that will be used to secure the pgwatch console and also grafana.

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+echo "Bringing down the project"
+
+
+docker-compose down
+
+
+for i in grafana influx logs/ pg pw2;
+do
+	chmod -R 777 $i;
```
Comment (ghost) [inline start.sh:11]: Remove the semi-colon at the end

**Review by ghost — APPROVED** [review]
All good - check the comment and the script for suggestion but nothing with impact

File: start.sh (hunk 1 above)
Comment (ghost) [reply start.sh:11]: Right here @MusaZwane

Comment (MusaZwane) [reply start.sh:11]: > Right here @MusaZwane

Please let me know which line?

File: docker-compose.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ version: '2.3'
 services:
  pgwatch:
   image: cybertec/pgwatch2:$BUILDTAG
+  restart: always
```
Comment (ghost) [inline docker-compose.yml:5]: Why `always` cos I know we normally use `unless-stopped`? Is there a specific reason for this change?

**Review by ghost — COMMENTED** [review]
See comments on code section

File: docker-compose.yml (hunk 2 above)
Comment (ghost) [reply docker-compose.yml:5]: @MusaZwane - here

**Pull request merged**